assets/app/view/tiles_page.rb

```diff
@@ -246,5 +254,108 @@ def render_toggle_button
         h(:'button.small', { on: { click: toggle } }, "Tile Names #{setting_for(@hide_tile_names) ? '❌' : '✅'}"),
       ])
     end
+
+    def render_test_tiles
+      # see /lib/engine/test_tiles.rb
+      test_tiles = Engine::TestTiles::TEST_TILES
+
+      scale = 2.0
+
+      return h(:div, [h(:p, 'Loading...')]) unless @connection
+
+      rendered_test_tiles = []
+
+      test_tiles.each do |title, fixtures|
+        if title
+          game_class = load_game_class(title)
+        else
+          fixtures[nil][nil].each do |hex_or_tile, opts|
+            %i[flat pointy].map do |layout_|
```

Comment:
why are you calling map here? what are you mapping to?

Reply:
In an earlier version I was probably returning the rendered stuff directly instead of `concat`-ing to `rendered_test_tiles`. Changed to `each`.